src/components/map-view.tsx

```diff
@@ -12,15 +12,15 @@ import { HurricaneTrack } from "./hurricane-track";
 import { LandfallRectangle } from "./landfall-rectangle";
 import { PrecipitationLayer } from "./precipitation-layer";
 import config from "../config";
-import { stores } from "../index";
 import CenterFocusStrong from "@material-ui/icons/CenterFocusStrong";
 import Home from "@material-ui/icons/Home";
 import { mapLayer } from "../map-layer-tiles";
 import { StormSurgeOverlay } from "./storm-surge-overlay";
 import { log } from "@concord-consortium/lara-interactive-api";
-
+import { LeafletMouseEvent } from "leaflet";
 import * as css from "./map-view.scss";
 import "leaflet/dist/leaflet.css";
+import { ThermometerMarker } from "./thermometer-marker";
```

Comment:
Conventionally, `.css` import would be last.

Reply:
Automatic imports in VS Code. :wink: Fixed in the last commit.